packages/babel-core/src/tools/build-external-helpers.ts

```diff
@@ -177,7 +177,11 @@ function buildHelpers(
     helpers.ensure(name, File);
     const { nodes } = helpers.get(name, getHelperReference, ref);
 
-    body.push(...nodes);
+    body.push(
+      expressionStatement(
+        callExpression(functionExpression(null, [], blockStatement(nodes)), []),
+      ),
+    );
```

Comment:
When bundling `babelHelpers`, we used to merge all the functions declarations in a single scope. This doesn't work when multiple helpers have different internal functions with the same name, which is the cause with the new `applyDecs2203` helper and `applyDecs`. This fixes it by wrapping them in IIFEs.